examples/se_2_3_localization.cpp

```diff
@@ -0,0 +1,302 @@
+/**
+ * \file se_2_3_localization.cpp
+ *
+ *  Created on: Aug 03, 2020
+ *     \author: prashanthr05
+ *
+ *  ---------------------------------------------------------
+ *  This file is:
+ *  (c) 2018 Joan Sola @ IRI-CSIC, Barcelona, Catalonia
+ *
+ *  This file is part of `manif`, a C++ template-only library
+ *  for Lie theory targeted at estimation for robotics.
+ *  Manif is:
+ *  (c) 2018 Jeremie Deray @ IRI-UPC, Barcelona
+ *  ---------------------------------------------------------
```

Comment:
This authorship should be updated

Reply:
Fixed in https://github.com/artivis/manif/pull/154/commits/945c4728639519d7eaa7bb47e7de7eacb9171fab